# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/script/moz.build
# Commit: e3fe772947bf
# Full Hash: e3fe772947bfdb692141089bdb3a314f4723afd9
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-18 21:52:29
# Regressor Bug: 1742437
# File Overlap Count: 1
# Description:
#   Bug 1742437 - Create DOM specific Module Loader; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D137298
# ==============================================================================

diff -r 467f757b6580 -r e3fe772947bf dom/script/moz.build
--- a/dom/script/moz.build	Fri Feb 18 12:45:51 2022 +0000
+++ b/dom/script/moz.build	Fri Feb 18 12:45:51 2022 +0000
@@ -20,7 +20,7 @@
 EXPORTS.mozilla.dom += [
     "AutoEntryScript.h",
     "LoadedScript.h",
-    "ModuleLoader.h",
+    "ModuleLoaderBase.h",
     "ModuleMapKey.h",
     "ScriptDecoding.h",
     "ScriptElement.h",
@@ -33,7 +33,7 @@
 UNIFIED_SOURCES += [
     "AutoEntryScript.cpp",
     "LoadedScript.cpp",
-    "ModuleLoader.cpp",
+    "ModuleLoaderBase.cpp",
     "ModuleLoadRequest.cpp",
     "ModuleMapKey.cpp",
     "nsIScriptElement.cpp",
